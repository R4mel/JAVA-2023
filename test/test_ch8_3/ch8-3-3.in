1
17
6
10
12
13
12
15
persons.txt
15
p.txt
7
0
6
0
0
0
