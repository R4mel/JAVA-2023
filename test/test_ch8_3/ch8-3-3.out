
************* Main Menu **************
* 0.exit 1.PersonManager 2.ch2 3.ch3 *
* 4.ch5 5.PMbyMap 6.MyVectorTest     *
**************************************
menu item? 1
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 17

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 6
-----------------
[data] directory: 0 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 10
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
-----------------
6 entries

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 12
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
[6] Kimm10 1948 54.4 false :Incheon Dong-gu: Kia CFO
[7] Park11 1095 98.9 true :Seoul Nam-gu: Computer 1.2 1
[8] Choi12 1882 72.8 false :Seoul Jung-gu: Computer 1.1 1
[9] Moon13 1960 81.2 false :Seoul Dong-gu:
[10] Nahh14 1646 93.9 false :Daejeon Buk-gu:
[11] Jooh15 1451 61.5 true :Daejeon Dong-gu:
[12] Moon16 1220 98.2 false :Gwangju Seo-gu:
[13] Kimm17 1668 64.8 false :Seoul Buk-gu: Samsung CFO
[14] Kang18 1067 40.8 true :Gwangju Dong-gu:
[15] Leem19 1290 48.6 false :Gwangju Jung-gu:
-----------------
16 entries

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 13
[0] persons.txt          F 2023-11-15 07:13       800
-----------------
[data] directory: 1 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 12
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
[6] Kimm10 1948 54.4 false :Incheon Dong-gu: Kia CFO
[7] Park11 1095 98.9 true :Seoul Nam-gu: Computer 1.2 1
[8] Choi12 1882 72.8 false :Seoul Jung-gu: Computer 1.1 1
[9] Moon13 1960 81.2 false :Seoul Dong-gu:
[10] Nahh14 1646 93.9 false :Daejeon Buk-gu:
[11] Jooh15 1451 61.5 true :Daejeon Dong-gu:
[12] Moon16 1220 98.2 false :Gwangju Seo-gu:
[13] Kimm17 1668 64.8 false :Seoul Buk-gu: Samsung CFO
[14] Kang18 1067 40.8 true :Gwangju Dong-gu:
[15] Leem19 1290 48.6 false :Gwangju Jung-gu:
[16] Yang20 1753 91.1 true :Incheon Jung-gu: Kia CFO
[17] Choi21 1577 40.1 true :Gwangju Jung-gu:
[18] Nahh22 1958 55.3 false :Incheon Jung-gu: Samsung Chief
[19] Shin23 1098 95.5 false :Busan Buk-gu:
[20] Song24 1901 92.4 true :Daegu Buk-gu: Electronics 0.6 4
[21] Park25 1380 69.4 false :Busan Jung-gu: Physics 2.1 2
[22] Johh26 1362 82.0 false :Seoul Dong-gu: Biology 2.1 3
[23] Kimm27 1317 74.8 true :Daejeon Jung-gu:
[24] Shim28 1113 64.5 true :Incheon Jung-gu: Computer 1.5 3
[25] Jooh29 1050 42.4 false :Busan Seo-gu:
-----------------
26 entries

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 15
[0] persons.txt          F 2023-11-15 07:13       800
-----------------
[data] directory: 1 files
new text file name to save? persons.txt
persons.txt: already exists

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 15
[0] persons.txt          F 2023-11-15 07:13       800
-----------------
[data] directory: 1 files
new text file name to save? p.txt
[0] p.txt                F 2023-11-15 07:13      1305
[1] persons.txt          F 2023-11-15 07:13       800
-----------------
[data] directory: 2 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 7
[0] p.txt                F 2023-11-15 07:13      1305
[1] persons.txt          F 2023-11-15 07:13       800
-----------------
[data] directory: 2 files
index number of a file to display [-1:stop]? 0
-----------------
P p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
P p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
S s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
S s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
W w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
W w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
W Kimm10 1948 54.4 false :Incheon Dong-gu: Kia CFO
S Park11 1095 98.9 true :Seoul Nam-gu: Computer 1.2 1
S Choi12 1882 72.8 false :Seoul Jung-gu: Computer 1.1 1
P Moon13 1960 81.2 false :Seoul Dong-gu:
P Nahh14 1646 93.9 false :Daejeon Buk-gu:
P Jooh15 1451 61.5 true :Daejeon Dong-gu:
P Moon16 1220 98.2 false :Gwangju Seo-gu:
W Kimm17 1668 64.8 false :Seoul Buk-gu: Samsung CFO
P Kang18 1067 40.8 true :Gwangju Dong-gu:
P Leem19 1290 48.6 false :Gwangju Jung-gu:
W Yang20 1753 91.1 true :Incheon Jung-gu: Kia CFO
P Choi21 1577 40.1 true :Gwangju Jung-gu:
W Nahh22 1958 55.3 false :Incheon Jung-gu: Samsung Chief
P Shin23 1098 95.5 false :Busan Buk-gu:
S Song24 1901 92.4 true :Daegu Buk-gu: Electronics 0.6 4
S Park25 1380 69.4 false :Busan Jung-gu: Physics 2.1 2
S Johh26 1362 82.0 false :Seoul Dong-gu: Biology 2.1 3
P Kimm27 1317 74.8 true :Daejeon Jung-gu:
S Shim28 1113 64.5 true :Incheon Jung-gu: Computer 1.5 3
P Jooh29 1050 42.4 false :Busan Seo-gu:
-----------------

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 6
-----------------
[data] directory: 0 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 0

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 0

************* Main Menu **************
* 0.exit 1.PersonManager 2.ch2 3.ch3 *
* 4.ch5 5.PMbyMap 6.MyVectorTest     *
**************************************
menu item? 0

Good bye!!
